src/wallet/rpcwallet.cpp

```diff
@@ -70,14 +70,14 @@ std::shared_ptr<CWallet> GetWalletForJSONRPCRequest(const JSONRPCRequest& reques
     return wallets.size() == 1 || (request.fHelp && wallets.size() > 0) ? wallets[0] : nullptr;
 }
 
-std::string HelpRequiringPassphrase(CWallet * const pwallet)
+std::string HelpRequiringPassphrase(const CWallet* pwallet)
```

Comment:
Why are these losing pointer constness?

Reply:
We don't really use pointer constness anywhere else in the code base. If it is required, the function should take a const reference instead.

Reply:
@luke-jr It is not the pointer losing constness, but the `pwallet` local variable losing constness.

`CWallet * const pwallet` means "Take a pointer-to-CWallet as argument, and store in the local variable `pwallet` which is const". It's not actually part of the function signature.

Reply:
It's not part of the function signature, but it safeguards against accidentally changing the pointer within the function.